### PR TITLE
dropdown and slider

### PR DESCRIPTION
**Pull request by preethasri**
created dropdown and slider components 

Screenshot (6)

Screenshot (7)
Screenshot (8)

### CODE REVIEW

**Review by techieshubham30 — APPROVED** [review]
Looks good to merge.

**Review by kshitijthatte — APPROVED** [review]
Looks good. 
Just one suggestion apart from screenshots, try adding the deploy preview link as well.

File: components/dropdownmenu/dropdownmenu.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+           style="width: 640px; height: 486px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (manojasarsa) [inline components/dropdownmenu/dropdownmenu.html:148]: Instead of inline styling, you can give a class for it, to be consistent with code.

**Review by manojasarsa — APPROVED** [review]
Code looks good. Just a small suggestion, remove extra blank lines.
Good to merge!

**Pull request merged**